common/src/main/java/com/wynntils/wynn/model/map/poi/CombatKind.java

```diff
@@ -13,7 +13,9 @@ public enum CombatKind {
     DUNGEONS("Dungeons", Texture.DUNGEON_ENTRANCE),
     GRIND_SPOTS("Grind Spots", Texture.GRIND_SPOT),
     RAIDS("Raids", Texture.RAID_ENTRANCE),
-    RUNE_SHRINES("Rune Shrines", Texture.RAID_ENTRANCE);
+    // TODO: We only have sprites for Uth Shrines, so those are the only ones that we have
+    // location markers for. Split up the category when more textures are available.
```

Comment:
If you are not happy with Uth shrine markers representing all shrines, then I'd rather say we need to update the texture to look more generic. I think we should avoid going down the route with having too specific markers.

Reply:
https://static.wikia.nocookie.net/wynncraft_gamepedia_en/images/2/20/WynnIconCSS.png/revision/latest?cb=20210928042006?v=20221206024249 has all runes as textures. You just need to crop them. I hope it is not too low quality. Although, if you check, our texture is a combination of UTH and AZ (color and shape). So it is "generic".

Reply:
Oh, that's interesting

Reply:
To clarify the genericness, maybe we should design one with the same look-and-feel, but a furthark rune R?

Reply:
Here's a design or what I mean: https://imgur.com/a/jVs8ZYY